packages/gatsby-theme/src/blocks/BlogHeadBlock/index.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from "prop-types"
+import BlogHead from "../../components/BlogHead"
```

Comment:
Please add BlogHead component to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js) and update this import
```suggestion
import { BlogHead } from "ui"
```